docs/install-upgrade/install.md

```diff
@@ -84,7 +94,36 @@ This control node should be given a static IP address. Either a lease or statica
 
 ### Configure Management Network
 
-The control node is dual-homed. It has a 10GbE interface that connects to the management network. The other link called `external` in the `fab.yaml` file is for the customer to access the control node. The management network is for the command and control of the switches that comprise the fabric. This management network can be a simple broadcast domain with layer 2 connectivity. The control node will run a DHCP and small http servers. The management network is not accessible to machines or devices not associated with the fabric.
+The control node is dual-homed. It has a 10GbE interface that connects to the management network. The other link called `external` in the `fab.yaml` file is for the customer to access the control node via their building IT network. The management network is for the command and control of the switches that comprise the fabric. This management network can be a simple broadcast domain with layer 2 connectivity. The control node will run a DHCP and small http servers. The management network is not accessible to machines or devices not associated with the fabric. The following diagram illustrates 3 networks for the control node. A "Local IT"  network that is used for command and control or IPMI. The other network is exclusive to Hedgehog, the best solution is a dedicated network switch, but a private exclusive vlan is potentially workable.
```

Comment:
```suggestion
The control node is dual-homed. It has a 10GbE interface that connects to the management network. The other link called `external` in the `fab.yaml` file is for the customer to access the control node via their building IT network. The management network is for the command and control of the switches that comprise the fabric. This management network can be a simple broadcast domain with layer 2 connectivity. The control node runs a DHCP and small HTTP servers. The management network is not accessible to machines or devices not associated with the fabric.

The following diagram illustrates 3 networks for the control node. A "Local IT" network that is used for command and control or IPMI. The other network is exclusive to Hedgehog, the best solution is a dedicated network switch, but a private exclusive VLAN is potentially workable.
```

What's the “building IT” network? The explanation in the following part says “local”, should we keep “local” in the paragraph to avoid confusion?

What is “the best solution”? What problem does this solution address?

> The following diagram illustrates 3 networks for the control node.

What's the third one? I see the local network and the Hedgehog-exclusive network, am I missing something?

Reply:
came here to comment the same thing, I also got confused by this paragraph

Reply:
I am going to remove the diagram. I don't think it is helping. Apologies for prematurely marking as resolved.